doc/user/content/third-party/supported-tools.md

```diff
@@ -1,33 +1,216 @@
+
+_Is there another DB management tool you'd like to use with Materialize? [Open a GitHub Issue here](https://github.com/MaterializeInc/materialize/issues/new?assignees=&labels=A-integration&template=02-feature.yml)._
+
+## Libraries and Drivers
+
+The following popular PostgreSQL libraries and drivers have been tested and confirmed to be working well with Materialize.
+
+| Service | Materialize Support | Notes |  |
+| --- | --- | --- | --- |
+| Node.js | 🟢 Production | The [`node-postgres` library](https://node-postgres.com/) can be used to [manage](/guides/node-js/#manage-sources-views-and-indexes), [query](/guides/node-js/#query) and even [stream](/guides/node-js/#stream) data from Materialize. | [More Info](/guides/node-js/) |
+| Python | 🟢 Production | The [`psycopg2` python package](https://pypi.org/project/psycopg2/) can be used to interact with Materialize as if it were a PostgreSQL DB. | |
+| Java | 🟢 Production | The popular [PostgreSQL JDBC driver](https://jdbc.postgresql.org/) can be used to interact with Materialize as if it were a PostgreSQL DB. |  |
+| Golang | 🟢 Production | TODO: Confirm which Go drivers have been tested. |  |
+| PHP | 🟢 Production | The standard PHP [PostgreSQL Extension](https://www.php.net/manual/en/ref.pgsql.php) can be used to interact with Materialize as if it were a PostgreSQL DB. |  |
+
+## Frameworks and ORMs
+
+Frameworks and ORMs tend to make more advanced queries to PostgreSQL behind the scenes, using configuration settings and system table endpoints that Materialize hasn't yet implemented. As Materialize [expands `pg_catalog` support](https://github.com/MaterializeInc/materialize/issues/2157), support for frameworks and ORMs will improve.
+
+| Service | Materialize Support | Notes |  |
+| --- | --- | --- | --- |
+| Ruby on Rails | 🔴 Researching | The Rails Active-Record ORM executes many PostgreSQL configuration queries and queries to `pg_catalog` endpoints that are not yet implemented in Materialize. | [](#notify) |
+| Prisma | 🔴 Researching | Prisma executes configuration queries and queries to `pg_catalog` endpoints that are not yet implemented in Materialize. | [](#notify) |
+
+_Is there another framework or ORM you'd like to use with Materialize? [Open a GitHub Issue here](https://github.com/MaterializeInc/materialize/issues/new?assignees=&labels=A-integration&template=02-feature.yml)._
+
+## Data Tools
+
+Many tools in the modern data stack can connect to Materialize via PostgreSQL, but like ORMs, these tools sometimes make advanced configuration and system table queries to endpoints that Materialize hasn't yet implemented.
+
+### Business Intelligence (BI) Tools
+
+| Service | Materialize Support | Notes |  |
+| --- | --- | --- | --- |
+| Metabase | 🟢 Beta | The Metabase PostgreSQL connector can be used to [connect Metabase to Materialize Core](/third-party/metabase/). | [More Info](/third-party/metabase/) [](#notify) |
+| Superset | 🟢 Beta | TBD @andrioni input. | [](#notify) |
```

Comment:
@andrioni Can you suggest a status and note for Superset here

Reply:
IMO Alpha until DevEx is able to take a deeper dive, and then beta. We have no known remaining issues with Superset and everything should work. Superset should support the TLS certs that we use for Cloud, but I haven't tested it yet.

Reply:
```suggestion
| Superset | 🟢 Alpha | Connect Superset to Materialize Core or Cloud using the Postgres connector.  | [](#notify) |
```